api/config/config_test.json

```diff
@@ -106,7 +106,7 @@
   },
   "rdrExport": {
     "host": "pmi-drc-api-test.appspot.com",
-    "queueName": "rdrQueueTest",
+    "queueName": "rdrQueue",
```

Comment:
Just confirming: I double-checked our cloud config, and I do see the new queue named rdrQueue. The rdrQueueTest queue is marked as disabled.

Reply:
Ohh my bad! That was because i accidentally updated queue name while testing the script.